velox/functions/lib/tests/RepeatTest.cpp

```diff
@@ -25,6 +27,16 @@ namespace {
 
 class RepeatTest : public FunctionBaseTest {
  protected:
+  static void SetUpTestCase() {
+    FunctionBaseTest::SetUpTestCase();
+    exec::registerStatefulVectorFunction(
+        "repeat", functions::repeatSignatures(), functions::makeRepeat);
+    exec::registerStatefulVectorFunction(
+        "array_repeat",
```

Comment:
array_repeat sounds similar to repeat and doesn't clarify "allow negative count" semantic; perhaps, rename to repeat_allow_negative_count?

Reply:
@mbasmanova, yes, your proposed name is better. Just updated. Thanks!